### PR TITLE
Fix/  packaged products and merch not showing in menu

### PR DESCRIPTION
**Pull request by jayant13o6**
## Description
- [x] merch and packaged products not shown in menu
- [x] additional text is to long

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Changes and Fixes
- Extra condition added to check productOptions length
- text length more than 50 will be replaced by '...'
 
## Screenshots 
image

## Checklist
- [ ] Database schema is updated
- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Looks good on large screens
- [ ] Looks good on mobiles
- [ ] Looks good on Tablets

Affected Apps
- [ ] Store
    - [ ] SEO
    - [x] Menu
    - [ ] Checkout
    - [ ] Add to Cart logic
    - [ ] Location Logic
    - [ ] Product page
    - [ ] Login
    - [ ] Payment
    - [ ] Kiosk

### CODE REVIEW

File: store/src/components/product_card.jsx [hunk 1]
```diff
@@ -334,8 +334,13 @@ export const ProductCard = props => {
                         </div>
                      ))}
                   {showProductAdditionalText && data?.additionalText && (
-                     <div className="hern-product-card__additional-text">
-                        {data.additionalText}
+                     <div className="hern-product-card__additional-text"
+                        title = {data?.additionalText}
```
Comment (rahul72925) [inline store/src/components/product_card.jsx:338]: if product has `additionalText` then show only `additionalText` otherwise show description.

Comment (jayant13o6) [reply store/src/components/product_card.jsx:338]: done

Comment (rahul72925) [reply store/src/components/product_card.jsx:338]: Are we using same `ProductCard` component on Product page to show product?

Comment (jayant13o6) [reply store/src/components/product_card.jsx:338]: same component

**Review by rahul72925 — APPROVED** [review]

**Review by shubhamz950 — APPROVED** [review]

**Pull request merged**